README.md

```diff
@@ -592,7 +592,9 @@ These items are relevant *only* on Windows.
 
 - Access the Windows clipboard through `/dev/clipboard`.
 
-- Run `cygstart` to open an arbitrary file through its registered application.
+- Use `cygstart` to open aa application(for example `cygstart calc` ) or an arbitrary file/directory through its registered application.
```

Comment:
there is a typo, this I think this may not a one of the most important tips for users, so I decide close it